### ISSUE TITLE
Some cells are not showing on jupyer notebook

### ISSUE THREAD

**Issue by promitbasak** [microsoft/vscode#130712]
Issue Type: <b>Bug</b>

I just opened the project after a while, opened a specific jupyter notebook file and found that most of the cells are not showing on vscode. I have checked code folding and other possible silly reasons that can happen, but the cells were just disappeared. 
Jupyter extension is installed on my vscode and I work with jupyter notebooks almost everyday.
I opened the file with text editor on vscode, and this shows all the cells. I also opened the file with the jupyter notebook program and all the cells werevisible to me. 
I tried to restart my vscode. Sometimes it works, I see all the cells with jupter extension. Sometimes it does not work and I do not see all my cells. So I have to restart my vscode for indefinite time until all the cells are visible.
This problem happens at times with some specific files, other files work just fine.

Extension version: 2021.8.1195043623
VS Code version: Code 1.59.0 (379476f0e13988d90fab105c5c19e7abc8b1dea8, 2021-08-04T23:13:12.822Z)
OS version: Windows_NT x64 10.0.19041
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz (4 x 2712)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.88GB (1.69GB free)|
|Process Argv|--crash-reporter-id ab59e581-5017-4846-b711-ce36e159cf57|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspyt653:30270858
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
pythonvspyt639:30300192
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
pythonvsuse255:30340121
vscod805cf:30301675
pythonvspyt200:30340761
vscextlangct:30333562
binariesv615:30325510
pythonvssor306:30344512
bridge0708:30335490
vstre464:30350172
bridge0723:30346310

```

</details>

**Comment by greazer** [comment]
Thanks for the feedback, @promitbasak! We'll take a look. However, I suspect that you may be running into this problem. https://github.com/microsoft/vscode/issues/130637. Once that's fixed, I'd be interested to know if you're still seeing this problem happen. Note that you'll need to try in VS Code Insiders.

**Comment by rebornix** [comment]
That should be fixed along with #130637 and available now in Insiders, please let us know how it works, thanks in advance!
